builtin/logical/mssql/path_config_lease.go

```diff
@@ -78,7 +87,7 @@ func (b *backend) pathConfigLeaseRead(
 	return &logical.Response{
 		Data: map[string]interface{}{
 			"ttl":     leaseConfig.TTL.String(),
-			"ttl_max": leaseConfig.TTLMax.String(),
+			"max_ttl": leaseConfig.TTLMax.String(),
```

Comment:
@chrishoffman Retain the old field in the response, along with the new one and add a warning to the response that the old one will be removed after 2 releases.